### PR TITLE
fix ReactiveCocoa/ReactiveCocoa#1869 rac_valuesAndChangesForKeyPath does...

### PR DESCRIPTION
**Pull request by larryonoff**
fix for ReactiveCocoa/ReactiveCocoa#1869

### CODE REVIEW

File: ReactiveCocoa/NSObject+RACPropertySubscribing.h [hunk 1]
```diff
@@ -72,9 +72,8 @@
 
 /// Creates a signal to observe the changes of the given key path.
 ///
-/// The initial value is sent on subscription, the subsequent values are sent
-/// from whichever thread the change occured on, even if it doesn't have a valid
-/// scheduler.
+/// The subsequent values are sent from whichever thread the change occured on,
+/// even if it doesn't have a valid scheduler.
```
Comment (jspahrsummers) [inline ReactiveCocoa/NSObject+RACPropertySubscribing.h]: How about:

> If there is an initial notification, it is sent upon subscription. Subsequent values are sent…

**Comment by jspahrsummers** [comment]
Sorry for the delay in reviewing this! Just one note. :pray:

File: ReactiveCocoa/NSObject+RACPropertySubscribing.h (hunk 1 above)
Comment (larryonoff) [inline ReactiveCocoa/NSObject+RACPropertySubscribing.h]: What do you think about:

> The initial value is sent on subscription if `NSKeyValueObservingOptionInitial` is set. The subsequent values are sent...

Comment (jspahrsummers) [inline ReactiveCocoa/NSObject+RACPropertySubscribing.h]: :+1:

**Comment by kastiglione** [comment]
@larryonoff Updates to pull requests don't generate notifications. Emoji's are the way to go.

Looks good, I'm sure @jspahrsummers won't mind if I merge this :stuck_out_tongue_winking_eye:

**Comment by larryonoff** [comment]
@kastiglione Thank you! I didn't know this.

**Pull request merged**